mautic/mautic
# Fix numeric reports table colums

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
I noticed there is wrong link in the reports table. This PR fixed it

![image](https://user-images.githubusercontent.com/462477/100875376-38f92680-34a6-11eb-80f4-69c88501df6a.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Make some  assets downloads
3. Then create assets report with Most downloaded assets
4. Check the number of downloads is pointed to weird url , for example http://0.0.0.1/
5. After PR, that column is not converted to clickable link
